lib/contracts/builtin_contracts.rb

```diff
@@ -201,6 +201,17 @@ def to_s
     end
   end
 
+  # Takes a list of values, e.g. +[:a, :b, :c]A+. If argument is included in the list, the contract passes.
```

Comment:
Can you break the comment at 80 characters?

Can you add an example like this `# Example: <tt>Enum[:a, :b, :c]</tt>`?